src/server/editionsPage.tsx

```diff
@@ -102,7 +115,16 @@ function render(
 	const item = fromCapi({ docParser, salt: imageSalt })(request);
 	const body = renderBody(item);
 	const thirdPartyEmbeds = getThirdPartyEmbeds(request.content);
-	const head = renderHead(request, thirdPartyEmbeds, body.css, body.ids);
+	const inlineStyles = requiresInlineStyles(request.content);
+
+	const head = renderHead(
+		item,
+		request,
+		thirdPartyEmbeds,
+		body.css,
+		body.ids,
+		inlineStyles,
```

Comment:
I think you can just pass this as `false`, the Editions version of the CSP ignores it anyway.

Reply:
Cool, fixed.